packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -199,6 +205,43 @@ enum ListTileControlAffinity {
 /// ```
 /// {@end-tool}
 ///
+/// For accessibility purposes, tappable leading and trailing widgets have to
+/// be at least 48x48px in size. However, to adhere with Material spec,
+/// trailing/leading widgets in one-line ListTiles should visually be at
+/// most 32px (dense: true) or 40px (dense: false) in height, which may
+/// conflict with the accessibility requirement.
+///
+/// For this reason, ListTile is implemented such that the height of leading
```

Comment:
We're only referring to one-line ListTiles here, right?

Reply:
Yes! I'll add that clarification